script/release/update-version-all.sh

```diff
@@ -143,6 +143,15 @@ for repository in `cat ${scriptDir}/../repository-list.txt` ; do
             sed -i "s/<version.org.kie>$kieOldVersion<\/version.org.kie>/<version.org.kie>$newVersion<\/version.org.kie>/" pom.xml
             returnCode=$?
 
+        elif [ "$repository" == "optaplanner" ]; then
+            mvnVersionsUpdateParentAndChildModules
+            returnCode=$?
+            if [ $returnCode != 0 ] ; then
+                exit $returnCode
+            fi
+            optaplanner-quickstarts/update-version.sh $newVersion $releaseType
```

Comment:
... or optaplanner-quickstarts/update-version.sh $newVersion community
since productized is not used any more! Both cases should work. So what you like more.

Reply:
Let's just pass it along, for consistency if you do ever end up using it again?

In any case, the other script currently doesn't do anything with it.